tox.ini

```diff
@@ -50,7 +50,8 @@ deps =
     devdeps: git+https://github.com/spacetelescope/gwcs.git
     devdeps: git+https://github.com/asdf-format/asdf.git
     devdeps: git+https://github.com/astropy/asdf-astropy.git
-    devdeps: git+https://github.com/spacetelescope/stdatamodels.git
+    # FIXME: https://github.com/spacetelescope/stdatamodels/issues/159
```

Comment:
this is tracked by #2165 on our end, right?

Reply:
Sure, but the canonical issue is upstream.